client/ui/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="zh-CN">
```

Comment:
Please use `<html lang="en">`. May be we should support Chinese and Japanese latter.

Reply:
ok.i will fix it.